dotnetv3/S3/ServerAccessLoggingExample/ServerAccessLoggingExample/ServerAccessLogging.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace ServerAccessLoggingExample
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    /// <summary>
+    /// This example shows how to enable logging on an Amazon Simple Storage
+    /// Service (Amazon S3) bucket. You need to have two Amazon S3 buckets for
+    /// this example. The first is the bucket for which you wish to enable
+    /// logging, and the second is the location where you want to store the
+    /// logs. The example was created using the AWS SDK for .NET version 3.7
+    /// and .NET Core 5.0.
+    /// </summary>
+    public class ServerAccessLogging
+    {
+        public static async Task Main()
+        {
+            string bucketName = "doc-example-bucket";
+            string logBucketName = "doc-example-logs-bucket";
+            string logObjectKeyPrefix = "Logs";
+
+            // If the AWS region defined for your default user is different
+            // from the AWS regsion where your Amazon S3 bucket is located,
+            // pass the AWS Region name to the S3 client object's constructor.
+            // For example, RegionEndpoint.USWest2 or RegionEndpoint.USEast2.
+            IAmazonS3 client = new AmazonS3Client();
+
+            try
+            {
+                // Grant Log Delivery group permission to write log entries
+                // to the target bucket.
+                await GrantPermissionsToWriteLogsAsync(client, logBucketName);
+
+                // Enable logging on the source bucket.
+                await EnableLoggingAsync(client, bucketName, logBucketName, logObjectKeyPrefix);
+            }
+            catch (AmazonS3Exception e)
+            {
+                Console.WriteLine($"Error: {e.Message}");
+            }
+        }
+
+        /// <summary>
+        /// This method grants appropriate permissions for logging to the
+        /// S3 bucket where the logs will be stored.
```

Comment:
S3 -> Amazon S3. I won't repeat this comment.

Reply:
S3 is correct in this instance. First time Amazon Simple Storage Service (Amazon S3), 2nd time Amazon S3, and third time only S3.